src/facebook_to_engagement_db/facebook_to_engagement_db.py

```diff
@@ -159,21 +167,34 @@ def _fetch_and_sync_facebook_to_engagement_db(google_cloud_credentials_file_path
                 if "parent" not in comment:
                     comment["parent"] = {}
 
-                origin_id = f'facebook_comment_id_{comment["id"]}'
-                message = _facebook_comment_to_engagement_db_message(comment, dataset.engagement_db_dataset,
-                                                                     origin_id, uuid_table)
+                #Only try to add the db comments that were created after the last seen comment.created_time
+                #This helps us reduce the number of reads to the db when checking for existing comments.
+                add_comment_to_db = True
+                if latest_comment_timestamp is not None and isoparse(comment['created_time']) <= latest_comment_timestamp:
+                    add_comment_to_db = False
+
+                if add_comment_to_db:
+                    origin_id = f'facebook_comment_id_{comment["id"]}'
+                    message = _facebook_comment_to_engagement_db_message(comment, dataset.engagement_db_dataset,
+                                                                         origin_id, uuid_table)
 
-                message_origin_details = {
-                    "page_id": facebook_source.page_id,
-                    "post_id": post_id,
-                    "user_id": comment["from"]["id"],
-                    "comment_id": comment["id"],
-                }
+                    message_origin_details = {
+                        "page_id": facebook_source.page_id,
+                        "post_id": post_id,
+                        "user_id": comment["from"]["id"],
+                        "comment_id": comment["id"],
+                    }
 
-                _ensure_engagement_db_has_comment(engagement_db, message, message_origin_details)
+                    _ensure_engagement_db_has_comment(engagement_db, message, message_origin_details)
 
+                    if cache is not None:
+                        cache.set_latest_comment_timestamp(post_id, isoparse(comment['created_time']))
+                else:
+                    log.info(f'Comment synced in previous run skipping ...')
```

Comment:
Can you tweak this to be
```
if not add_comment_db:
    continue

# Add the comment to the db
origin_id = ...
...
```

In otherwords, keeping the mainline logic unindented